support-frontend/assets/pages/supporter-plus-thank-you/components/thankYouHeader/heading.tsx

```diff
@@ -193,13 +197,23 @@ function Heading({
 	amount,
 	currency,
 	contributionType,
+	ratePlanKey,
 	paymentStatus,
 	promotion,
 }: HeadingProps): JSX.Element {
 	const isPending = paymentStatus === 'pending';
 	const isDigitalEdition = productKey === 'DigitalSubscription';
 	const isGuardianAdLite = productKey === 'GuardianAdLite';
 	const isTier3 = productKey === 'TierThree';
+	const isPaperOrGuardianWeekly = [
+		'NationalDelivery',
+		'HomeDelivery',
+		'GuardianWeeklyDomestic',
+		'GuardianWeeklyRestOfWorld',
+		'SubscriptionCard',
+	]
+		.map((product) => productKey === product)
+		.includes(true);
```

Comment:
We could simplify this just using includes directly
```suggestion
	const isPaperOrGuardianWeekly = [
		'NationalDelivery',
		'HomeDelivery',
		'GuardianWeeklyDomestic',
		'GuardianWeeklyRestOfWorld',
		'SubscriptionCard',
	].includes(productKey);
```